Source/Flow/Public/Nodes/Route/FlowNode_CustomOutput.h

```diff
@@ -16,6 +16,9 @@ class FLOW_API UFlowNode_CustomOutput final : public UFlowNode
 
 	UPROPERTY()
 	FName EventName;
+
+	UPROPERTY(EditAnywhere, Category = "Output")
+	bool bFinish;
```

Comment:
A new property in this form is currently a mystery for a user. It needs a more descriptive name (`bFinishGraph` perhaps?) and comment above it. This way a random user could use it right away :)